test/runtests.sh

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env bash
 
+# change to script directory
+cd "${0%/*}"
```

Comment:
What's the behavior of this line? Change to wherever folder the script is located?